lib/providers/apns.js

```diff
@@ -83,7 +84,7 @@ ApnsProvider.prototype._setupPushConnection = function(options) {
   connection.on('socketError', errorHandler);
 
   connection.on('transmissionError', function(code, notification, recipient) {
-    var err = new Error('Cannot send APNS notification: ' + code);
+    var err = new Error(g.f('Cannot send {{APNS}} notification: %s', code));
```

Comment:
I think you have to run `slt-globalize -e` again for `intl/en/messages.json` to get updated again with the brackets change.